test/LibraryManager.Test/VersionedLibraryNamingSchemeTest.cs

```diff
@@ -12,10 +12,15 @@ public class VersionedLibraryNamingSchemeTest
         [DataTestMethod]
         [DataRow("jquery@3.3.1", "jquery", "3.3.1")]
         [DataRow("@angular/cli@1.0.0", "@angular/cli", "1.0.0")]
+        [DataRow("@angular/cli@", "@angular/cli", "")]
+        [DataRow("@angular/cli", "@angular/cli", "")]
```

Comment:
`@angular/cli`, `@angular/`, and `@angular` are all redundant with other cases already specified in this list.  Arguably, so is `@angular/cli@`.

Reply:
Fixed in commit 2. I think I'll leave the `@angular/cli@` as it proves that this case has been fixed, even though it is technically redundant

Reply:
Sure, that sounds good.